src/AlgebraicGeometry/Schemes/PrincipalOpenSubset/Objects/Attributes.jl

```diff
@@ -27,7 +27,7 @@ function inclusion_morphism(U::PrincipalOpenSubset; check::Bool=false)
   if !isdefined(U, :inc)
     X = ambient_scheme(U)
     inc = SpecMor(U, X, hom(OO(X), OO(U), gens(OO(U)), check=check), check=check)
-    U.inc = OpenInclusion(inc, ideal(OO(X), complement_equation(U)), check=check)
+    U.inc = PrincipalOpenEmbedding(inc, [complement_equation(U)], check=check)
```

Comment:
It seems that `PrincipalOpenEmbedding` somewhat competes with `OpenInclusion`?
Can this lead to problems later?

Reply:
`OpenInclusion` does not require the image to be a principal open subset. I do not see how this entails conflicts, no.